src/main.cpp

```diff
@@ -556,14 +575,27 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // Enable Special Options ----------------------------------------------
+        if ( cmdLine.GetFlagArgument ( i, CMDLN_SPECIAL ) )
```

Comment:
After thinking a bit more about the new style: I think it would still be good to see the options in the code like before. But I agree that the current way is not good since it leads to a lot of unneeded code. 

Something like a function getting all arguments from the CLI and returning an array of enabled options (or similar) which we loop through to set the options would be another option.

Reply:
Please explain further...
The code isn't that different from how it was, except for the defines for the short + long names. Is that what you have a problem with ? 
Personally I think it makes the code a lot more readable, and it has another reason... since it is no longer necessary to pass command line parameters as variable true a lot of constructors, this could mean the long and short names could spread throughout the code and that didn't seem a good idea to me, so I use defines instead.

And I think there is hardly any more code, it's just in another place.